olp-cpp-sdk-dataservice-read/src/repositories/PartitionsCacheRepository.cpp

```diff
@@ -327,6 +333,40 @@ bool PartitionsCacheRepository::ContainsTree(
   return cache_->Contains(CreateQuadKey(key, depth, version));
 }
 
+cache::KeyValueCache::KeyListType
+PartitionsCacheRepository::GetRelatedPartitionKeys(
```

Comment:
It is rather a `CreatePartitionKeys()` as you create them rather than getting them from the cache.